.github/workflows/clear_cloudflare_cache.yml

```diff
@@ -0,0 +1,12 @@
+name: Clear cloudflare cache 

```

Comment:
> To run a workflow manually, the workflow must be configured to run on the workflow_dispatch event.

https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#configuring-a-workflow-to-run-manually